### ISSUE TITLE
Validating ebs volume type fails for instances which don't have EBS support

### ISSUE THREAD

**Issue by ayushmpb95** [terraform-providers/terraform-provider-aws#13867]
### Description

I'm creating/updating AWS ES using terraform and has been using aws provider to validate the templates. However, with upgradation to version `2.67.0` modules have started failing up.

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version
**v0.11.14**
### Affected Resource(s)

* aws_elasticsearch

### Terraform Configuration Files

```
provider "aws" {
  version = "2.67.0"
  region  = "us-west-2"

  assume_role {
    role_arn = "arn:aws:iam::abc"
  }
}

```

```
  ...
  instance_type         = "i3.xlarge.elasticsearch"
  instance_count        = "3"
 ....

```

### Panic Output

```
Error: module.abc.aws_elasticsearch_domain.es: expected ebs_options.0.volume_type to be one of [standard gp2 io1], got 
```

### Expected Behavior

Nothing to fail since the instances doesn't support EBS.

### Actual Behavior

```
Error: module.abc.aws_elasticsearch_domain.es: expected ebs_options.0.volume_type to be one of [standard gp2 io1], got 
```

### Steps to Reproduce

Install version `2.67.0` of  terraform aws provider and try to run aws elasticsearch template with instances which don't support EBSes.

### References

https://github.com/terraform-providers/terraform-provider-aws/releases/tag/v2.67.0

**Comment by anGie44** [comment]
Relates #12668

**Comment by ewbankkit** [comment]
@ayushmpb95 Thanks for raising this issue.
In your `aws_elasticsearch_domain` configuration, do you have `volume_type = ""` or no `volume_type` set at all?

**Comment by ayushmpb95** [comment]
It is the former i.e. `volume_type = "" `

```
volume_type = "${var.volume_size > 0 ? local.volume_type : ""}"
```

**Comment by maxblaze** [comment]
This is also breaking for us, we also have `volume_type = ""`

**Comment by anGie44** [comment]
A fix has been merged and will release with `v2.68.0` of the Terraform AWS Provider, expected out later today.

**Comment by ayushmpb95** [comment]
Thank you so much for fixing this.
